sdk/search/test-resources.json

```diff
@@ -43,7 +43,7 @@
         },
         "searchSku": {
             "type": "string",
-            "defaultValue": "free",
+            "defaultValue": "basic",
```

Comment:
This gives me a tiny bit of pause given how much and often we run.  It would be good to look at costs before/after and maybe split out a separate test project for features that require basic.

Reply:
It should be a drop in the bucket compared to what other services are costing, but this would probably best be solved by weekly tests, which EngSys is working on now. @benbp is working on changes that will allow us to pass ARM parameters, and in a weekly test we could opt into Key Vault and set the search SKU. It's necessary to test Key Vault since "free" doesn't support CMKs (wasn't in the docs, but I opened one and it seems to have gotten fixed).

I opened https://github.com/Azure/azure-sdk-for-net/issues/16576 to track.